github.md

```diff
@@ -22,3 +32,73 @@ you should do:
 ```bash
 $ git merge <branch-name> --no-ff
 ```
+
+### Pull Request Reviews
+
+In most (all) cases, pull requests should be reviewed by at least one other developer, even if it is just for syntax or raise general questions in the case of unfamiliarity with a particular language. 
+
+When creating a new pull request, you need to pick what branch you're merging - usually your dev branch - and what you're aiming to merge into - usually master, create a logical title, and add a description that includes: 
```

Comment:
When opening a PR from a single commit, the message copies automatically into the PR message... so make a great commit message to begin with and you get a great PR message for free ;)